tests/mainnet/state/serialization/contract_bytecode_key.yaml

```diff
@@ -4,6 +4,7 @@
 # https://github.com/ethereum/portal-network-specs/blob/638aca50c913a749d0d762264d9a4ac72f1a9966/state-network-test-vectors.md
 
 address: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+address_hash: '0x8679e8eda65bd257638cf8cf09b8238888947cc3c0bea2aa2cc3f1c4ac7a3002'
```

Comment:
nit: `address_hash` seems redundant, considering that we already have `address`. 

I would remove `address_hash` (since it can be derived from the `address`).

Also in the other file.